# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webcodecs/EncoderTypes.h
# Commit: b1577155be1c
# Full Hash: b1577155be1c9e9070c91a969c0cdcfaab9c8936
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2023-12-23 09:52:03
# Regressor Bug: 1749047
# File Overlap Count: 1
# Description:
#   Bug 1749047 - harzar fix. CLOSED TREE
#   
#   
#   This patch uses RootedDictionary around `EncodedVideoChunkMetadata`,
#   `VideoDecoderConfig` and `VideoColorSpaceInit` in `OutputEncodedData` to
# ==============================================================================

diff -r ecf1c8556163 -r b1577155be1c dom/media/webcodecs/EncoderTypes.h
--- a/dom/media/webcodecs/EncoderTypes.h	Sat Dec 23 10:32:48 2023 +0200
+++ b/dom/media/webcodecs/EncoderTypes.h	Sat Dec 23 08:17:11 2023 +0000
@@ -77,7 +77,7 @@
   using ConfigTypeInternal = VideoEncoderConfigInternal;
   using InputType = dom::VideoFrame;
   using InputTypeInternal = mozilla::VideoData;
-  using OutputConfigType = mozilla::dom::VideoDecoderConfig;
+  using OutputConfigType = mozilla::dom::VideoDecoderConfigInternal;
   using OutputType = EncodedVideoChunk;
   using OutputCallbackType = EncodedVideoChunkOutputCallback;
   using MetadataType = EncodedVideoChunkMetadata;